### DISCUSSION TITLE
line-clamp doesn't work properly on safari

### DISCUSSION THREAD

**Discussion by sandeshtamanq** [tailwindlabs/tailwindcss#9086] [Help] [answered]
Line clap doesn't truncate the paragraph properly on safari

**Comment by thecrypticace** [comment]
Hey I am not seeing an issue here in Safari. Can you please provide more details on what you're experiencing?

Here's an example: https://play.tailwindcss.com/182J2l7rnr

And what I see:
<img width="1308" alt="Screen Shot 2022-08-12 at 12 08 15" src="https://user-images.githubusercontent.com/614993/184397208-fff48f84-7dc3-4159-843c-18e77387e580.png">

Which looks correct to me. Safari has had some bugs around line-clamp and interactions like [this one](https://bugs.webkit.org/show_bug.cgi?id=240410) where changing text color and removing line-clamp while hovering causes text to disappear. If you can provide a reproduction on what you're experiencing and what you expect to see instead we should be able to figure out how to help.

**Comment by nthpaul** [comment, marked as answer]
If you know what height you want the text to stay within, then you can do something like `h-{value} line-clamp-{value} overflow-hidden`

**Comment by li5tun** [comment]
in Safari, ellipsis do not place on the last line.

**Comment by li5tun** [comment]
in Safari, ellipsis do not place on the last line. the cause might be fixed height, you can try to git rid of `h-{value}`
